examples/ini.js

```diff
@@ -0,0 +1,47 @@
+const fs = require("fs");
```

Comment:
Please use single quotes:

```suggestion
const fs = require('fs');
```